api/src/entity/media.entity.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+} from "typeorm";
+
+@Entity("media")
+export class MediaEntity {
+    @PrimaryGeneratedColumn()
+    Id: number;
+
+    @Column()
+    ExternalId: number;
```

Comment:
Entendo que o ID é externo porque vem da API, mas acho que seria mais intuitivo se chamássemos ele de id aqui também

Reply:
Não entendi. Como assim chamar ele de Id?
Você diz, mesclar o Id com o ExternalId?

Se for isso, acho que é uma má prática a gente usar como chave primária um ID que não está no nosso controle. Se fosse uma aplicação real, o que aconteceria se de repente a gente quisesse mudar de fonte? Ou mesmo ter uma base própria?

Acabaríamos ficando muito acoplados ao TMDB

Reply:
Eu não tinha visto o `id` em cima kkkk
Acho que está ótimo assim mesmo